public/css/index.css

```diff
@@ -27,76 +49,70 @@ body {
 
 .form__input {
   background: none;
-  border: #2a324e solid 1px;
-  border-radius: 15px;
-  padding: 13px;
+  border: var(--form-input-main-border-color) solid var(--basic-border-size);
+  border-radius: var(--form-input-radius);
+  padding: 16px;
   color: white;
-  width: 80%;
 }
 
 .form__input:focus {
-  border-color: #fb00ff;
-  background: #1d253b;
+  border-color: var(--form-input-focus-color);
+  background: var(--form-input-focus-bg-color);
   outline: none;
 }
 
 .form__field {
   display: flex;
   flex-direction: column;
+  width: 80%;
   margin-bottom: 40px;
-  position: relative;
 }
 
 .btn {
   text-align: center;
-  padding: 10px 50px;
-  border-radius: 10px;
-  position: absolute;
+  cursor: pointer;
+  padding: 16px 48px;
+  max-width: 150px;
+  border-radius: var(--button-border-radius);
   color: white;
-  border: 1px solid gray;
+  border: var(--basic-border-size) solid var(--button-main-color);
 }
 
 .btn__field {
   background: none;
   border: none;
-  width: 10%;
+  padding: 16px;
   cursor: pointer;
-  position: absolute;
-  right: 5%;
-  top: 50%;
 }
 
 .btn-blue {
-  background: #305cf4;
-  bottom: 20px;
+  background: var(--button-blue-color);
 }
 
 .btn-pink {
-  background: #d54fd7;
-  bottom: 20px;
-  right: 20px;
+  background: var(--button-pink-color);
 }
 
-@media (min-width: 768px) {
-  .form {
-    width: 100%;
-  }
-  .btn-blue {
-    bottom: 40%;
-  }
+.btn-pink-position-controller {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.btn-wrapper {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  height: 50vh;
```

Comment:
czemu 50vh? raczej powinno brać pod uwagę kontener w którym jest, a nie rozmiar ekranu. I jeśli chciałbyś, żeby wypełniał całą pozostałą przestrzeń na dole formularza, to istnieją inne sposoby raczej na to